mod/quiz/locallib.php

```diff
@@ -1563,7 +1563,10 @@ function quiz_send_confirmation($recipient, $a) {
  * @return int|false as for {@link message_send()}.
  */
 function quiz_send_notification($recipient, $submitter, $a) {
-    global $PAGE;
+    global $PAGE, $DB;
+
+    $submitterentries = $DB->get_record('user_info_data', array('userid' => $submitter->id, 'fieldid' => '1'));
```

Comment:
fieldID = 1 will be always department? it will be more accurate if we search field by like name